### PR TITLE
moving to yarn

### PR DESCRIPTION
**Pull request by alexandercbooth**
🐈

### CODE REVIEW

File: packages/transform-dataresource/README.md [hunk 1]
```diff
@@ -14,15 +14,12 @@ _Note: the desktop app can be used instead, but you'll have to manually reload t
...
+yarn
 ```
 
 ### 2. Setup Jupyter Extension
 ___Note: this requires Python >= 3.6___
 
-First, install lerna globally
-```bash
-npm install -g lerna
 ```
```
Comment (rgbkrk) [inline packages/transform-dataresource/README.md:23]: I think the backticks above here need to be deleted.

Comment (rgbkrk) [reply packages/transform-dataresource/README.md:23]: Oh this was merged, I'll make a PR. 😄

Comment (rgbkrk) [reply packages/transform-dataresource/README.md:23]: https://github.com/nteract/nteract/pull/3172

Comment (alexandercbooth) [reply packages/transform-dataresource/README.md:23]: Oops, sorry about that 😬!

Comment (alexandercbooth) [reply packages/transform-dataresource/README.md:23]: I tried to move too fast 😅

Comment (rgbkrk) [reply packages/transform-dataresource/README.md:23]: Hey all good, better to have fixes than staying stale. We can always keep adjusting docs. 😄

**Pull request merged**